src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -509,6 +509,7 @@ private void UnzipPackage(string filePath, string scriptPath)
                 _logger.LogInformation($"Extracting files to '{scriptPath}'");
                 ZipFile.ExtractToDirectory(filePath, scriptPath, overwriteFiles: true);
                 _logger.LogInformation($"Zip extraction complete");
+                RunBashCommand($"chmod -R 755 '{scriptPath}'", MetricEventNames.LinuxContainerSpecializationChModCommand);
```

Comment:
This can impact cold start since it'll recursively enumerate the full app content

Reply:
That's a good point. Instead of scoping the fix just for HttpWorker, I am looking into options to enable users to execute any binary from the function code. Let me know if you have any ideas on how to achieve this without impacting cold start.

Reply:
Thanks for pointer on `Mono.Poxis`. Scoping this change to HttpWorker for now.